R/run_trial.R

```diff
@@ -124,9 +124,9 @@ run_trial <- function(
       .options = furrr::furrr_options(seed = seed)
     )
   } else {
-    res <- lapply(
-      X = sim_ids,
-      FUN = f
+    res <- purrr::map(
```

Comment:
just cleanup. In principle should do exactly the same, but purrrr should be closer to furrr than lapply.